react/react1/week3/todo-list-app/src/components/TodoList.js

```diff
@@ -0,0 +1,28 @@
+import TodoItem from './TodoItem'
+import Wrapper from './Wrapper'
+
+const TodoList = (props) => {
+  return (
+    <>
+      {props.todos.length ? (
+        props.todos.map((todo) => {
+          return (
+            <Wrapper key={todo.id}>
+              <TodoItem
+                todo={todo}
+                key={todo.id}
+                deleteTodoById={props.deleteTodoById}
+                setTodoAsCompleted={props.setTodoAsCompleted}
+                changeDescription={props.changeDescription}
```

Comment:
Would recommend using a context in the future (although the task wasn't about it), otherwise - good job!